gradle.properties

```diff
@@ -1,4 +1,4 @@
 org.gradle.caching=true
 group=org.ballerinalang.azure_storage_service
-version=0.1.2-SNAPSHOT
+version=0.1.3-SNAPSHOT
```

Comment:
Please dont change the connector version in the master branch. Leave it as it is.